docs/reference/reference-deploy/config/master-config-reference.rst

```diff
@@ -343,6 +343,22 @@ The master supports the following configuration settings:
          compute resources, e.g. GPUs or dedicated CPUs. Defaults to the Slurm/PBS default partition
          if it has GPU resources and if no resource pool is specified.
 
+      -  ``job_project_source``: configures labelling of jobs on the HPC cluster (via Slurm
+         ``--wckey`` or PBS ``-P``). Allowed values are:
+
+         -  ``project``: Use the project name of the experiment (this is the default, if no project
+            nothing is passed to workload manager).
+
+         -  ``workspace``: Use the workspace name of the project (if no workspace, nothing is passed
+            to workload manager).
+
+         -  ``label[:prefix]``: Use the value from the experiment configuration labels list (if no
+            matching labels, nothing is passed to workload manager). If a label begins with prefix,
+            remove it and use the remainder as the value for the WCKey/Project. If multiple values
+            begin with prefix, the values are concatenated together with a comma (,) separator on
+            Slurm or underscore (_) with PBS. If the prefix is not specified or empty, all labels
+            will be matched (and therefore concatenated).
```

Comment:
Workload managers do not generally support multiple WCKey/Project values so it is recommended that ``prefix`` is configured to match a single label to enable use of the workload manager reporting tools that summarize usage by each WCKey/Project value.

Reply:
DOne